green_cli/authenticators/__init__.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, options):
 
     @property
     def _mnemonic(self):
-        return self.mnemonic_prop.get()
+        return self.mnemonic_prop.get().strip()
```

Comment:
I think this is trying to do the same thing as `SoftwareAuthenticator.set_mnemonic` which does a split/join, I wonder if there should be a `MnemonicOnDisk.normalise` or some such that is called by the getter and the setter.

Reply:
Updated to normalize all the things